models/job_models.go

```diff
@@ -0,0 +1,81 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+//go:generate reform
+
+// EchoJobResult stores echo job specific result data.
+type EchoJobResult struct {
+	Message string `json:"message"`
+}
+
+// JobType represents job type.
+type JobType string
+
+// Supported job types.
+const (
+	Echo = JobType("echo")
+)
+
+// JobResult describes a job result which is storing in persistent storage.
+//reform:job_results
+type JobResult struct {
+	ID         string    `reform:"id,pk"`
+	PMMAgentID string    `reform:"pmm_agent_id"`
+	Type       JobType   `reform:"type"`
+	Done       bool      `reform:"done"`
+	Error      string    `reform:"error"`
+	Result     []byte    `reform:"result"`
```

Comment:
I don't think that we should store job result here.

Reply:
What is your proposal? Separate table?

Reply:
Probably we will have separate tables for each job type(backup, restore). Here we can collect logs output.
let's keep `Result` for now, later we will decide where keep job results.